Logos/cisco-logo-72px.svg

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="72px" height="72px" viewBox="0 0 72 72" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: Sketch 52.6 (67491) - http://www.bohemiancoding.com/sketch -->
+    <desc>Created with Sketch.</desc>
+    <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
+        <g transform="translate(0.000000, 15.000000)" fill="#049FD9">
+            <polygon id="Fill-1" points="21 28 24 28 24 42 21 42"></polygon>
+            <path d="M50,32.0833333 C49.8837209,31.9666667 48.8372093,31.3833333 47.2093023,31.3833333 C45.1162791,31.3833333 43.6046512,32.9 43.6046512,35 C43.6046512,36.9833333 45,38.6166667 47.2093023,38.6166667 C48.7209302,38.6166667 49.8837209,38.0333333 50,37.9166667 L50,41.5333333 C49.5348837,41.65 48.4883721,42 46.9767442,42 C43.255814,42 40,39.3166667 40,35 C40,31.0333333 42.9069767,28 46.9767442,28 C48.4883721,28 49.6511628,28.35 50,28.4666667 L50,32.0833333" id="Fill-2"></path>
+            <path d="M16,32.0833333 C15.8837209,31.9666667 14.8372093,31.3833333 13.2093023,31.3833333 C11.1162791,31.3833333 9.60465116,32.9 9.60465116,35 C9.60465116,36.9833333 11,38.6166667 13.2093023,38.6166667 C14.7209302,38.6166667 15.8837209,38.0333333 16,37.9166667 L16,41.5333333 C15.5348837,41.65 14.4883721,42 12.9767442,42 C9.25581395,42 6,39.3166667 6,35 C6,31.0333333 8.90697674,28 12.9767442,28 C14.4883721,28 15.6511628,28.35 16,28.4666667 L16,32.0833333" id="Fill-3"></path>
```

Comment:
Also please replace all of the `Fill-x` ids with [unique GUIDs ](https://www.guidgenerator.com), or remove them entirely if they are not needed